src/util/tubemap.js

```diff
@@ -3145,14 +3153,28 @@ function drawNodes(dNodes, groupNode) {
     .on("mouseout", nodeMouseOut)
     .on("dblclick", nodeDoubleClick)
     .on("click", nodeSingleClick)
-    .style("fill", config.transparentNodesFlag ? "none" : "#fff")
-    .style("fill-opacity", config.showExonsFlag ? "0.4" : "0.6")
-    .style("stroke", "black")
+    .style("fill", (d) => colorNodes(d.name)["fill"])
+    .style("fill-opacity", (d) => colorNodes(d.name)["fill-opacity"])
+    .style("stroke", (d) => colorNodes(d.name)["outline"])
     .style("stroke-width", "2px")
     .append("svg:title")
     .text((d) => getPopUpNodeText(d));
 }
 
+function colorNodes(nodeName){
```

Comment:
We should explain the new function.

```suggestion
// Given a node name, return an object with "fill", "fill-opacity", and "outline"
// keys describing what colors should be used to draw it.
function colorNodes(nodeName){
```